src/App.js

```diff
@@ -328,7 +300,24 @@ function App() {
 }
 
 function Root() {
-  return (
+  const [initializing, setInitializing] = React.useState(true);
+
+  React.useEffect(() => {
+    async function initializeApplication() {
+      await initSentry(); // must be set up immediately
+
+      const deviceId = await getDeviceId();
+
+      Sentry.setUser({ id: deviceId });
+      // Segment.identify(deviceId)
```

Comment:
why are we leaving this in?

Reply:
Sorry, #4202 is coming in right after this, was just leaving this here to showcase what else I have planned for this codeblock.